addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4119,7 +4119,7 @@ export class OdooEditor extends EventTarget {
         }
     }
 
-    cleanForSave(element = this.editable) {
+    cleanForSave(element = this.editable, editable) {
```

Comment:
If you're going to introduce an `editable` argument, make sure it's used consistently in the function. [Below](https://github.com/odoo/odoo/pull/143737/files#diff-c309d19284e2e1f38a1dba1d68e0262371686595f3668a3bc0f1ca35e3189a0bR4177), `if (this.deselectTable() && hasValidSelection(this.editable)) {` should use that argument when it can (Github won't let me comment on that line because you didn't change it).

Also, to make that less verbose, I suggest to change this line to:
```suggestion
    cleanForSave(element = this.editable, editable = this.editable) {
```